### ISSUE TITLE
clusterbats: error in reporting errors

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#589]
**Version:** r7_login_image 1b0aad3
**Environment:** tmaster

When running updatenode node001 cv_fly_clusterbats i got the following report:
```
node001: Fri Jan 15 09:31:10 UTC 2016 Running postscript: cv_fly_clusterbats
node001: Installed Bats to /usr/local/bin/bats
node001: 1..19
node001: ok 1 bash executes
node001: ok 2 1.1.4 controller connects to internet
node001: ok 3 1.1.5 firewall is disabled
node001: ok 4 1.1.6 SElinux is disabled
node001: ok 5 1.1.7 The timezone is set correctly
node001: ok 6 1.1.8 Hostname is set correctly
node001: ok 7 1.1.9 The controller node is setup to user LDAP for authentication
node001: ok 8 1.1.10 The controller node hosts a docker registry with a trinity image
node001: ok 9 1.1.11 DNS is working on the controller
node001: ok 10 1.1.13 Openvswitch is available
node001: ok 11 1.1.14 The controller hosts an openstack image
node001: ok 12 1.2.0.a We can configure the switch
node001: ok 13 # skip 1.2.1 We can discover compute nodes
node001: ok 14 1.2.5 We can assign the containers to the default virtual cluster a
node001: ok 15 1.2.6 There is a virtual login node
node001: ok 16 1.2.7 Slurm and munge are running on the virtual login nodes
node001: not ok 17 1.2.8 The compute nodes can connect to the internet
node001: # (in test file /trinity/testing/clusterbats/t1.2.bats, line 84)
node001: #   `ssh -o StrictHostKeyChecking=no node001 ping -c5 8.8.8.8' failed with status 255
node001: # ssh: connect to host node001 port 22: Connection refused

node001: ok 18 /cluster/vc-a/.modulespath is a file not a directory
node001: not ok 19 3.2.2 Slurm report the correct containers for our cluster
node001: # (in test file /trinity/testing/clusterbats/t3.2.bats, line 5)
node001: #   `echo $output | grep -F "idle"' failed
node001: 1..1
node001: ok 1 #579 docker logs: nslcd entered FATAL state
node001: Postscript: cv_fly_clusterbats exited with code 0
node001: Running of postscripts has completed.
```

However, when logging in to check 1.2.8: 
```
root@controller(tmaster) ~]# ssh node002
Warning: Permanently added 'node002,10.141.0.2' (ECDSA) to the list of known hosts.
[root@node002 ~]# docker ps
CONTAINER ID        IMAGE                     COMMAND                  CREATED             STATUS              PORTS               NAMES
9c82e2f6e176        controller:5050/trinity   "/usr/bin/supervisord"   2 minutes ago       Up 2 minutes                            trinity
[root@node002 ~]# ping google.com
PING google.com (74.125.136.100) 56(84) bytes of data.
64 bytes from ea-in-f100.1e100.net (74.125.136.100): icmp_seq=1 ttl=48 time=5.47 ms
64 bytes from ea-in-f100.1e100.net (74.125.136.100): icmp_seq=2 ttl=48 time=5.47 ms
```

The compute nodes were connected to the internet. 

3.2.2 was also ok on manual inspection:

```
[root@login ~]# sinfo
PARTITION   AVAIL  TIMELIMIT  NODES  STATE NODELIST
containers*    up   infinite      2   idle c[001-002]
```

Maybe bats flew too quickly :)

**Comment by hansthen** [comment]
Fixed in b1a6a469d.

**Comment by themisa** [comment]
Fixed, tested in:

[root@controller(tmaster) ~]# /trinity/report
**Version:** r7 b55e0e1
